4985_CommAudio/server.cpp

```diff
@@ -3,9 +3,159 @@
 #include <map>
 #include <WS2tcpip.h>
 
+QAudioFormat format;
```

Comment:
Does this need to be global? It looks like you only use it in one function, why not make it local to that function?